ci/gpu/build.sh

```diff
@@ -47,8 +47,9 @@ conda install -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge \
     "rapids-build-env=$MINOR_VERSION.*"
 
 # https://docs.rapids.ai/maintainers/depmgmt/ 
-# conda remove -f rapids-build-env rapids-notebook-env
-# conda install "your-pkg=1.0.0"
+logger "Conda install custom..."
+conda remove --force rapids-build-env rapids-notebook-env
+conda install -c conda-forge scipy
```

Comment:
Since (I think) you're trying to install `scipy>=1.5`, don't you need to specify `>=1.5`? Otherwise, is it possible it would be satisfied with a lower version already installed?